src/main/java/com/redhat/cloud/policies/app/rest/PolicyCrudService.java

```diff
@@ -381,14 +396,20 @@ public Response storePolicy(@QueryParam("alsoStore") boolean alsoStore, @NotNull
             return Response.status(Response.Status.FORBIDDEN).entity(new Msg("Missing permissions to store policy")).build();
         }
 
-        policy.store(user.getAccount(), policy);
-        accountLatestUpdateRepository.setLatestToNow(user.getAccount());
+        if (user.getOrgId() != null) {
+            policy.store(user.getOrgId(), policy);
+            accountLatestUpdateRepository.setLatestOrgIdToNow(user.getOrgId());
+        }
+        if (user.getAccount() != null) {
+            policy.store(user.getAccount(), policy);
+            accountLatestUpdateRepository.setLatestToNow(user.getAccount());
+        }
```

Comment:
There an issue here as it will store the same policy 2 times.

Reply:
fuck yes. We should add a technical debt task. There are so many tests missing everywhere.